src/main/resources/injection.st

```diff
@@ -0,0 +1,16 @@
+import org.springframework.util.AntPathMatcher
+import org.springframework.util.PathMatcher
+
+PathMatcher antMatcher = new AntPathMatcher("\0")
+
+noNominalV = <nominalV> == -1
+noCountries = <countries>.isEmpty()
+
+for (equipment in network.<collectionName>) {
+  if (   (noNominalV || equipment.terminal.voltageLevel.nominalV <nominalVOperator> <nominalV>)
+         && (antMatcher.match('<equipmentId>', equipment.id) || (equipment.optionalName.isPresent() && antMatcher.match('<equipmentName>', equipment.name)))
+         && (noCountries || equipment.terminals.stream().anyMatch{connectable ->
```

Comment:
Same we should expose a standard function to user to filter equipment by country: like isLocatedIn(equipment, countries)

Reply:
Done